chapters/memory-layout/introduction-to-GDB/drills/tasks/inspect/support/.gitignore

```diff
@@ -0,0 +1 @@
+/inspect
```

Comment:
If we're adding a .gitignore file I think `*.o` files should be ignored aswell, same for every task.

Reply:
They're already ignored by the `.gitignore` file in the root of the repository.

Reply:
`.o` files are excluded in the global `.gitignore` [1]. There's no need to exclude them here.

[1] https://github.com/cs-pub-ro/hardware-software-interface/blob/main/.gitignore#L2